.github/workflows/make_bundle_conda.yml

```diff
@@ -0,0 +1,359 @@
+
+      - name: Run build scripts (PySide) # filename under feedstock's .ci_support/
+        shell: bash -l {0}
+        working-directory: napari-feedstock
+        env:
+          DOCKER_IMAGE: quay.io/condaforge/linux-anvil-comp7
+          CONFIG: linux_64_qt_bindingspyside2
+          CI: github_actions
+        run: |
+          export CONDA_BLD_PATH="${{ github.workspace }}${{ env.conda_bld_suffix }}"
+          ./.scripts/run_docker_build.sh
+
+      - name: Export artifact identifier
+        id: artifact-id
+        run: |
+          name=pkgs-napari-${{ env.NAPARI_VERSION }}-${{ github.run_id }}-${{ github.run_number }}-${{ github.run_attempt }}
+          echo "::set-output name=name::${name}"
+
+      - name: Upload packagess as artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ steps.artifact-id.outputs.name }}
+          path: ${{ github.workspace }}${{ env.conda_bld_suffix }}noarch/*.tar.bz2
+
+      - name: Upload to anaconda.org
+        shell: bash -l {0}
+        # if: github.event_name == 'schedule'
+        run: |
+          anaconda \
+            -t ${{ secrets.ANACONDA_TOKEN }} \
+            upload \
+            --skip-existing \
+            -u napari \
+            -l nightly \
+            ${GITHUB_WORKSPACE}${conda_bld_suffix}noarch/*.tar.bz2
+
+  bundle:
+    name: Bundle ${{ matrix.target-platform }}
+    runs-on: ${{ matrix.os }}
+    needs: conda-build
+    if: github.repository == 'napari/napari'
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            python-version: "3.9"
+            target-platform: "linux-64"
+          - os: macos-latest
+            python-version: "3.9"
+            target-platform: "osx-64"
+          - os: macos-latest
+            python-version: "3.9"
+            target-platform: "osx-arm64"
+          - os: windows-latest
+            python-version: "3.8"
+            target-platform: "win-64"
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+      DISPLAY: ":99.0"
+      CONDA_BLD_PATH: ${{ github.workspace }}/conda-bld
+      CONSTRUCTOR_TARGET_PLATFORM: ${{ matrix.target-platform }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Create environment.yml
+        shell: bash
+        run: |
+          cat <<EOF > environment.yml
+          name: build-bundle
+          channels:
+            - napari/label/bundle_tools
+            - conda-forge
+          dependencies:
+            - python ${{ matrix.python-version }}
+            - pip
+            - constructor
+            - conda-build
+            - ruamel.yaml
+            - conda-standalone
+          EOF
+      - name: install micromamba
+        uses: mamba-org/provision-with-micromamba@main
+
+      - name: set up build dependencies
+        shell: bash -l {0}
+        run: |
+          python -m pip install -e '.[bundle_build]'
```

Comment:
We might need to update this section in setup.cfg to reflect what we actually need in this script, or merge requirements with the environment file defined above.

Reply:
Why not update setup.cfg? Seems like the simple solution. Is it so that both methods work simultaneously for some time? If so, then we could have two bundle_build sections?

Reply:
>Why not update setup.cfg? Seems like the simple solution.

👍🏼